packages/partysocket/src/index.ts

```diff
@@ -87,3 +87,4 @@ export default class PartySocket extends ReconnectingWebSocket {
 }
 
 export const WebSocket = ReconnectingWebSocket;
+export { ReconnectingWebSocket };
```

Comment:
ok I tested, and this works, and doesn't require the new export. 

```suggestion
export { ReconnectingWebSocket as WebSocket };
```